go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/whyrusleeping/yamux
+
+go 1.12
```

Comment:
that was probably auto-added when you built, but I don't think we mandate it yet.

Reply:
It doesn't really make a difference. It just means go will warn the user if the compile _fails_.

>  The go directive in a go.mod file now indicates the version of the language used by the files within that module. It will be set to the current release (go 1.12) if no existing version is present. If the go directive for a module specifies a version newer than the toolchain in use, the go command will attempt to build the packages regardless, and will note the mismatch only if that build fails.